geth/api/api.go

```diff
@@ -221,3 +221,15 @@ func (api *StatusAPI) NotifyUsers(message string, payload fcm.NotificationPayloa
 
 	return err
 }
+
+// ConnectionChange handles network state changes logic.
+func (api *StatusAPI) ConnectionChange(typ string, expensive bool) {
```

Comment:
Is it typo? `s/typ/type`

Reply:
typ is a short for `type` which is keyword, yes. It's a common naming for Go code (stdlib uses it a lot).

Reply:
I'm no friend of this convention, but I'm a friend of following standard conventions. So 👍